src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/ProjectInstalledPackages.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
```

Comment:
This ProjectInstalledPackages.cs change comes from Martin's [PR](https://github.com/NuGet/NuGet.Client/pull/3539). He already had PR for reference type unit tests and I'm using some of his code only for reference type project unit test.